src/History/SearchKeyword.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+
+function TotalKeywordButton({ Keyword = "..." }) {
```

Comment:
P1 : 매개변수가 string 이므로 소문자로 시작하는 것은 어떨까요?
```suggestion
function TotalKeywordButton({ keyword = "..." }) {
```

Reply:
위에 내용과 동일한 피드백인 것 같아 소문자로 모두 수정완료하였습니다 :)